### PR TITLE
feat: Fix alarm icon on header

### PR DESCRIPTION
**Pull request by juntaepark**

### CODE REVIEW

File: packages/web/src/components/common/notification-button/NotificationButton.styles.ts [hunk 1]
```diff
@@ -8,17 +8,20 @@ export const NotificationWrapper = styled.div`
...
+  ${media.tablet} {
+    width: 24px;
+    height: 24px;
+  }
   .notification-icon {
-    width: 36px;
-    height: 36px;
-    padding: 4px;
+    width: 21px;
+    height: 25.08333396911621px;
```
Comment (jinoosss) [inline packages/web/src/components/common/notification-button/NotificationButton.styles.ts:21]: How about rounding down the decimal point?

**Review by jinoosss — COMMENTED** [review]
I left one small comment.

File: packages/web/src/components/common/notification-button/NotificationButton.styles.ts (hunk 1 above)
Comment (juntaepark) [reply packages/web/src/components/common/notification-button/NotificationButton.styles.ts:21]: good idea.

**Review by jinoosss — APPROVED** [review]
Nicely done !

**Pull request merged**